src/hooks/useAnimationLoop/useAnimationLoop.stories.mdx

```diff
@@ -0,0 +1,61 @@
+import { Meta } from '@storybook/blocks';
+
+<Meta title="hooks/useAnimationLoop" />
+
+# useAnimationLoop
+
+useAnimationLoop hook is a wrapper around requestAnimationFrame method. This hook will execute a
+callback function every next frame.
+
+## Reference
+
+```ts
+function useAnimationLoop(callback: (delta: number) => void, enabled = false): void;
+```
+
+### Parameters
+
+- `callback` - function accepts `delta` parameter which represents time passed since last
+  invocation; - `callback` function will be called on every animation frame
```

Comment:
Is this formatting a bit strange? It's also mentioning `callback` twice.
It's a bit tricky to read (or maybe when rendered it's better)

Reply:
fixed